### ISSUE TITLE
trojan in file

### ISSUE THREAD

**Issue by Yout00** [DuncanRuns/MinecraftAutoResetter#2]
cant use MinecaftAutoResetter bc windows defender says there is a trojan in there

**Comment by LukeSaward1** [comment]
False-positive. If it contained a virus, it would not be on github.
